src/plone/restapi/services/discussion/configure.zcml

```diff
@@ -13,31 +13,31 @@
       method="GET"
       factory=".conversation.CommentsGet"
       for="*"
-      permission="zope2.View"
+      permission="plone.app.discussion.ViewComments"
```

Comment:
@razvanMiu @jensens I'm still not sure about this as it will introduce a hard-dependency on `plone.app.discussion >= 4.x`

Reply:
version wise probably it does. But OTHO its clean this way.

Reply:
@avoinea if we introduce such a hard dependency we need to reflect this in setup.py (with all the downsides that come with that). The alternative would be to use zcml conditional with a fallback (which increase the overall complexity). I need to find a free minute to think about this (not easy these days tbh).

Reply:
@tisto If you ask me, I would leave `zope2.View` in zcml, as we already check for `plone.app.discussion.ViewComments` in python code 

https://github.com/plone/plone.restapi/blob/611063c108488ec6b0c8b2b26f3327dd8bb98340/src/plone/restapi/serializer/discussion.py#L27-L29

https://github.com/plone/plone.restapi/blob/611063c108488ec6b0c8b2b26f3327dd8bb98340/src/plone/restapi/services/discussion/utils.py#L31-L35

Reply:
Given the current situation I am fine with `View` - but in a future release this should be changed. It is probably a good idea to create an issue to not forget about it.